llvm/include/llvm/CodeGen/SDPatternMatch.h

```diff
@@ -490,6 +490,13 @@ m_c_SetCC(const T0_P &LHS, const T1_P &RHS, const T2_P &CC) {
                                                          CC);
 }
 
+template <typename T0_P, typename T1_P, typename T2_P>
+inline TernaryOpc_match<T0_P, T1_P, T2_P, false, false>
```

Comment:
Is it necessary to pass the `false` values?

Reply:
updated